pyqtgraph/examples/CLIexample.py

```diff
@@ -1,20 +1,25 @@
 """
-Display a plot and an image with minimal setup. 
+CLI Example
```

Comment:
"Tutorialized" this example https://pyqtgraph--2323.org.readthedocs.build/en/2323/examples/CLIexample.html#sphx-glr-examples-cliexample-py

Reply:
Was wondering if this example even worked anymore now that interactive mode doesn't necessarily work with some bindings.

Reply:
That's a good point, looks like this usage currently only works with PyQt{5,6}. So maybe not the best example to iterate on, but I *believe* this would still work with PySide at least for the docs build. I use PySide2 for qtgallery and the [iterative example](https://qtgallery.readthedocs.io/en/latest/auto_examples/iterative.html) works.